site/content/community/meetings/_index.adoc

```diff
@@ -41,16 +41,16 @@ https://meet.google.com/pii-faxn-woh[Google Meet Link]
 |===
 | Date | Time
 
-| 2025-05-15 | 9:00 AM PST
+| 2025-06-26 | 9:00 AM PST
 18:00 CET
```

Comment:
CEST (look outside the window - it's summer, man ;) )

Reply:
Really ? it's summer ? Time is flying 😄